<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Layer_Preview</class>
 <widget class="QDialog" name="Layer_Preview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layer_Preview</string>
  </property>
  <widget class="QGroupBox" name="groupBoxSeed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>571</width>
     <height>311</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>600</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Seed Preview  </string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>31</y>
      <width>552</width>
      <height>334</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelSeed">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Seed Threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoadSeedPreview">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Load Seed for Preview</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelSeedValue">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Value: 0.01</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="thresholdSliderSeed">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>550</width>
         <height>15</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>550</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxGrow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>571</width>
     <height>311</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>600</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string>Grow Preview  </string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>31</y>
      <width>552</width>
      <height>334</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelGrow">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Grow Threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoadGrowPreview">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Load Grow for Preview</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelGrowValue">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Value: 0.01</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="thresholdSliderGrow">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>550</width>
         <height>15</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView_2">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>550</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>730</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>561</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Click for Choosing Background Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLoadBackground">
      <property name="text">
       <string>Load Background image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
